lib/David_tic_tac_toe/validation.rb

```diff
@@ -0,0 +1,23 @@
+module Validation
+  def self.validate_input(input)
+    input.gsub!(/[[:space:]]/, '')
+    user_input = input.split(',')
+    (user_input.length == 2) && user_input.all?(/[0-9]/)
+  end
+
+  def self.validate_position(index, board)
+    puts index
+    puts board
+    denote_empty = '_'
```

Comment:
Since this is already defined in the Board class, I would try to reference the constant from here instead of creating another one.

Reply:
1b8092c74aedfd9c1ae34fa91dee6692b08e5eb5